jack-core/src/com/rapleaf/jack/queries/QueryFetcher.java

```diff
@@ -26,13 +32,19 @@ public static Records getQueryResults(PreparedStatement preparedStatement, Set<C
       return results;
     } catch (SQLRecoverableException e) {
       dbConnection.resetConnection();
-      throw e;
+      throw new RuntimeException(e);
```

Comment:
I think some downstream code retries if it is a sql recoverable exception. Changing it to runtime exception will disable those retires.

Reply:
Oh, didn't mean to change this. Must have leaked from an earlier version.